packages/ndla-image-search/src/ImageSearch.js

```diff
@@ -331,9 +332,9 @@ class ImageSearch extends React.Component {
   }
 
   searchImages(queryObject) {
+    const { searchImages, onError } = this.props;
     this.setState({ searching: true });
-    this.props
-      .searchImages(queryObject.query, queryObject.page, this.props.locale)
+    searchImages(queryObject.query, queryObject.page, this.props.locale)
```

Comment:
hvorfor henter du ikke ut locale lenger opp?  const { searchImages, onError, locale } = this.props;